examples/todocontroller/e2e-test.sh

```diff
@@ -47,7 +47,24 @@ function create() {
 function check_msg() {
   bazel build examples/todocontroller:example-todo.describe
 
-  OUTPUT=$(./bazel-bin/examples/todocontroller/example-todo.describe)
+  NUM_ATTEMPTS=10
+  OUTPUT=""
+  for iattempt in $(seq 1 $NUM_ATTEMPTS); do
+    OUTPUT=$(./bazel-bin/examples/todocontroller/example-todo.describe)
+    echo "Attempt ${iattempt}/${NUM_ATTEMPTS}: Checking whether server is done with update"
+    # Disable errexit temporarily because we expect the following command to
+    # fail a few times
+    set +o errexit
+    echo "${OUTPUT}" | grep "Done:[ ]*true"
```

Comment:
Most of this can be simplified to something like: `echo $OUTPUT | grep Done && return 0 || true`